src/Confluent.SchemaRegistry.Serdes.Avro/SpecificDeserializerImpl.cs

```diff
@@ -53,7 +53,13 @@ public SpecificDeserializerImpl(ISchemaRegistryClient schemaRegistryClient)
 
             if (typeof(ISpecificRecord).IsAssignableFrom(typeof(T)))
             {
-                ReaderSchema = (global::Avro.Schema)typeof(T).GetField("_SCHEMA", BindingFlags.Public | BindingFlags.Static).GetValue(null);
+                var schemaField = typeof(T).GetField("_SCHEMA", BindingFlags.Public | BindingFlags.Static);
+                if (schemaField is null)
+                {
+                    throw new ArgumentException("Deserializable type must have public static field with name '_SCHEMA'.");
```

Comment:
I think we want more information, maybe something like: `($"ISpecificRecord implementation '{}' ...", typeof(T).FullName)` or something like that? The call stack will indicate this happened during deserialization, though that could also be in the message.

Reply:
Done